Modules/Core/Transform/test/itkBSplineTransformGTest.cxx

```diff
@@ -56,9 +57,9 @@ void bspline_eq( const itk::BSplineTransform<TParametersValueType, NDimensions,
     typename ImageType::Pointer coeffImage1 = bspline1->GetCoefficientImages()[i];
     typename ImageType::Pointer coeffImage2 = bspline2->GetCoefficientImages()[i];
 
-    EXPECT_EQ( coeffImage1->GetOrigin(), coeffImage2->GetOrigin() ) << description;
+    EXPECT_VECTOR_NEAR( coeffImage1->GetOrigin(), coeffImage2->GetOrigin(), tolerance ) << description;
     EXPECT_EQ( coeffImage1->GetDirection(), coeffImage2->GetDirection() ) << description;
```

Comment:
No tolerance for direction?

Reply:
There are two reasons for this:
 1. The directions are directly stored in the fixed params ( actually mesh size is too and could be EXPECT_EQ too).
2. The `EXPECT_VECTOR_NEAR` does not correctly work on the `Matrix` type.